docs/source/tutorial.rst

```diff
@@ -543,12 +543,38 @@ jobs.
 Resource Monitoring
 ===================
 JADE optionally monitors CPU, disk, memory, and network utilization
-statistics in structured log events. You can enable the feature by passing
-``-rX`` or ``--resource-monitor-interval=X`` where ``X`` is an interval in
-seconds.
+statistics in structured log events. There are two ways to collect statistics:
 
-If reports are enabled (in ``submit-jobs``) then JADE will save interactive
-plots of the this data in ``<output-dir>/stats``.
+1. ``aggregation``: Track average/min/max stats in memory on each node and
+   generate a summary report at the end (default).
+2. ``periodic``: Record stats in files on each node and generate a summary
+   report and interactive HTML plots at the end. The reports include
+   average/min/max stats.
+
+You can specify the mode in ``jade submit-jobs`` or ``jade config
+submitter-params``.
```

Comment:
Maybe specify the long and short versions of the argument here? It would be nice to see the `-R` syntax.